config/config.go

```diff
@@ -300,242 +301,244 @@ func NewConfig() *Config {
 	cfg.LuaModules = commaListFlag()
 	cfg.Oauth2GrantTokeninfoKeys = commaListFlag()
 
-	flag.StringVar(&cfg.ConfigFile, "config-file", "", "if provided the flags will be loaded/overwritten by the values on the file (yaml)")
+	flags := flag.NewFlagSet(progname, flag.ContinueOnError)
```

Comment:
I think we can use 
```go
flag := flag.NewFlagSet(progname, flag.ContinueOnError)
```
It will mask package name but will keep following flag definitions intact

Reply:
If it will only change in the function scope then I think it's ok but we meed to make sure it works